locales/en-US/app.ftl

```diff
@@ -20,6 +20,31 @@ AppHeader--app-header = <header>{ -profiler-brand-name }</header> — <subheader
 AppHeader--github-icon =
     .title = Go to our git repository (this opens in a new window)
 
+## AppViewRouter
+
+AppViewRouter--error-message-from-addon =
+    .message = Couldn't retrieve the profile from Firefox.
```

Comment:
- Use typographic apostrophe `’` instead of a straight single quote `'` (worth checking other strings).
- Use the Fluent term instead of Firefox.